h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -207,12 +117,174 @@ class SearchBarController extends Controller {
       }
     };
 
+    let explanationList = [
+      {
+        matchOn: 'user',
+        title: 'user:',
+        explanation: 'search by username',
+      },
+      {
+        matchOn: 'tag',
+        title: 'tag:',
+        explanation: 'search for annotations with a tag',
+      },
+      {
+        matchOn: 'url',
+        title: 'url:',
+        explanation: 'see all annotations on a page',
+      },
+      {
+        matchOn: 'group',
+        title: 'group:',
+        explanation: 'show annotations created in a group you are a member of',
+      },
+    ].map((item)=>{ return Object.assign(item, { type: FACET_TYPE}); });
+
+    // tagSuggestions are made available by the scoped template data.
+    // see search.html.jinja2 for definition
+    let tagSuggestions = window.h && window.h.tagSuggestions;
+
+    let tagsList = (tagSuggestions || []).map((item)=>{
+      return Object.assign(item, {
+        type: TAG_TYPE,
+        title: escapeHtml(item.tag), // make safe
+        matchOn: stringUtil.fold(stringUtil.normalize(item.tag)),
+        usageCount: item.count || 0,
+      });
+    });
+
+
+    this._suggestionsHandler = new AutosuggestDropdownController( this._input, {
+
+      list: [].concat(explanationList, tagsList),
```

Comment:
I can update. But I did this intentionally because I think it reads more easily and leaves the slight lack of tribal knowledge question "is tagsList is being added to the explanationList?" out of the way